features/FEATURE_BLE/ble/Gap.h

```diff
@@ -2293,7 +2571,9 @@ class Gap {
         const BLEProtocol::AddressBytes_t peerAddr,
         BLEProtocol::AddressType_t ownAddrType,
         const BLEProtocol::AddressBytes_t ownAddr,
-        const ConnectionParams_t *connectionParams
+        const ConnectionParams_t *connectionParams,
+        const uint8_t *peerResolvableAddr = NULL,
```

Comment:
why resolvable? isn't it just any random address?

Reply:
ah, it gets set in peerAddr I think

Reply:
Your right to asking, it requires updated comments: If peerAddrtype is equal to `PUBLIC_IDENTITY` or `RANDOM_STATIC_IDENTITY` the identity (resolved) address goes into peerAddr and the private resolvable address used by the peer goes into peerResolvableAddr.